app/viewmodels/ResultsPageSummary.scala

```diff
@@ -49,7 +53,23 @@ class ResultsPageSummary(from: LocalDate, to: LocalDate, day: Boolean = true)
         content = span(messages("cf.cash-account.detail.csv")),
         visuallyHiddenText = Some(messages("cf.cash-account.detail.csv-definition"))
       )))
+    )
+  }
 
+  private def cashTransactionsResultRowV2(dates: CashTransactionDates,
+                                          fullStop: Boolean,
+                                          day: Boolean): SummaryListRow = {
+    summaryListRow(
+      value = HtmlFormat.escape(
+        if (fullStop) {
+          rowResult(dates, day)
+        } else {
+          rowResultWithoutFullStop(dates, day)
+        }).toString(),
+      secondValue = None,
+      actions = Actions(items = Seq(ActionItem(
```

Comment:
It could be on the same lien as below as it is not making line long

actions = Actions(items = Seq(ActionItem(href = controllers.routes.SelectTransactionsController.onPageLoad().url

Reply:
with the closing parentheses in same line, its going beyond 120 characters, so gave a line break before items param